src/pages/Home/Home.jsx

```diff
@@ -58,10 +62,10 @@ const Home = () => {
           <SlArrowDown style={{ fontSize: '52px', color: '#FFFFFF' }} />
         </button>
       </div>
-      <div className="to-do" id="to-do">
+      <div className={styles['to-do']} id={styles['to-do']}>
```

Comment:
we'll rarely need a `className` and an `id` together on a component but in the future pls make sure they're not the same - if they are, ask if we really need both